cmd/ci-operator-configresolver/main.go

```diff
@@ -244,6 +328,7 @@ func main() {
 	http.HandleFunc("/", handler(http.HandlerFunc(http.NotFound)).ServeHTTP)
 	http.HandleFunc("/config", handler(resolveConfig(configAgent, registryAgent)).ServeHTTP)
 	http.HandleFunc("/resolve", handler(resolveLiteralConfig(registryAgent)).ServeHTTP)
+	http.HandleFunc("/step-registry", handler(resolveStepRegistry(registryAgent, configAgent)).ServeHTTP)
```

Comment:
could just be

```
/step
/chain
/workflow
```
?

Reply:
could be, but then, so could pkg/webreg, right?  Keeping this nested allows us to copy/paste the prefix handling out of pkg/webreg if/when that evolves so we keep the path structure consistent between the two hosts for these resources.

Reply:
and the job config is already not consistent. i don't see any utilty in this

Reply:
job config?  I'm just trying to minimize divergence from the existing path structure that webreg uses to identify config subsections.  Are you concerned about `jobHandler` being wired up to a different place in this config.ci host (other than the `/step-registry` namespacing)?